nemo/collections/tts/models/fastpitch.py

```diff
@@ -228,11 +228,10 @@ def tb_logger(self):
             if self.logger is None and self.logger.experiment is None:
                 return None
             tb_logger = self.logger.experiment
-            if isinstance(self.logger, LoggerCollection):
-                for logger in self.logger:
-                    if isinstance(logger, TensorBoardLogger):
-                        tb_logger = logger.experiment
-                        break
+            for logger in self.trainer.loggers:
+                if isinstance(logger, TensorBoardLogger):
+                    tb_logger = logger.experiment
```

Comment:
I think they deprecated LoggerCollection - should we upgrade and stop using it too ? Maybe just pass list of callback to trainer ?